hapi/src/services/hyperion/updaters/genesis-fundtransfer.updater.js

```diff
@@ -0,0 +1,45 @@
+const {
+  edenDelegatesGql,
+  edenElectionGql,
+  edenHistoricElectionGql
+} = require('../../../gql')
+
+module.exports = {
+  type: `genesis:fundtransfer`,
+  apply: async action => {
+    try {
+      const election = edenHistoricElectionGql.get(
+        action.data.distribution_time
+      )
+
+      let registeredMember = await edenDelegatesGql.get({
+        account: { _eq: action.data.to }
+      })
+
+      if (!registeredMember)
+        registeredMember = await edenDelegatesGql.save(memberData)
+
+      const electionData = {
+        id_delegate: registeredMember.id,
+        election_round: election.election_round,
+        delegate_level: action.data.rank
+      }
+
+      const registeredElection = await edenElectionGql.get({
+        id_delegate: { _eq: registeredMember.id },
+        election_round: { _eq: electionData.election_round }
+      })
+
+      if (!registeredElection) await edenElectionGql.save(electionData)
+
+      if (registeredElection.delegate_level < electionData.delegate_level)
```

Comment:
when you have multiple lines that belongs to an if statement, wrap it inside of `{}`